pkgs/development/python-modules/django-picklefield/default.nix

```
@@ -1,16 +1,25 @@
-{ lib, buildPythonPackage, fetchPypi, django }:
+{ lib, buildPythonPackage, fetchFromGitHub, django }:
 
 buildPythonPackage rec {
   pname = "django-picklefield";
   version = "3.0.1";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "15ccba592ca953b9edf9532e64640329cd47b136b7f8f10f2939caa5f9ce4287";
+  src = fetchFromGitHub {
+    owner = "gintas";
+    repo = "django-picklefield";
+    rev = "4d5e3b2995df6597d45ad59d92ead38abc5c34af";
+    sha256 = "0ni7bc86k0ra4pc8zv451pzlpkhs1nyil1sq9jdb4m2mib87b5fk";
   };
 
   propagatedBuildInputs = [ django ];
 
+  dontUseSetuptoolsCheck = true;
+
+  # Taken from project's tox.ini
+  checkPhase = ''
+    python3 -m django test -v3 --settings=tests.settings
```

to support alt interpreters such as pypy
```suggestion
    ${python.interpreter} -m django test -v3 --settings=tests.settings
```

---

Do I also need to add an import for this?

---

you would, if it's not already imported